### PR TITLE
(BIDS-2399) Change validator performance columns to numeric

### PR DESCRIPTION
**Pull request by MauserBitfly**
### <samp>🤖 Generated by Copilot at a890c22</samp>

This pull request adds validation and logging for the chain configuration values used in the statistics program, and changes the column types of the validator performance table to support fractional values. These changes enhance the reliability and accuracy of the statistics.

### CODE REVIEW

File: db/migrations/tbd_vlidator_performance_column_types.sql
Comment (Eisei24) [inline db/migrations/tbd_vlidator_performance_column_types.sql:1]: **typo:** **vlidator** in the file name. It has to be renamed for the "tbd" change anyway.

**Review by Eisei24 — APPROVED** [review]
lgtm

File: cmd/statistics/main.go [hunk 1]
```diff
@@ -62,6 +62,13 @@ func main() {
 	}
 	utils.Config = cfg
 
+	if utils.Config.Chain.Config.SlotsPerEpoch == 0 || utils.Config.Chain.Config.SecondsPerSlot == 0 {
+		logrus.Fatalf("error ether SlotsPerEpoch [%v] or SecondsPerSlot [%v] are not set", utils.Config.Chain.Config.SlotsPerEpoch, utils.Config.Chain.Config.SecondsPerSlot)
```
Comment (recy21) [inline cmd/statistics/main.go:66]: One should use utils.LogFatal instead.

Comment (MauserBitfly) [reply cmd/statistics/main.go]: LogFatal is now used

**Review by recy21 — APPROVED** [review]

**Pull request merged**